test-browser/testenv.py

```diff
@@ -0,0 +1,42 @@
+import os
+import datetime
+from selenium import webdriver
+
+DRIVER = None
+timestamp_str = datetime.datetime.now().strftime('%Y-%m-%dT%H:%M:%S%z')
+tunnel_name = 'custom'
+
+ci_travis = ("TRAVIS" in os.environ) and (os.environ["TRAVIS"] == "true")
+
+use_sauce = ci_travis
+use_sauce = True
```

Comment:
You could modify : `use_sauce = False`, then tests will be running in local Firefox, without external connections to SouceLabs. Useful for local testing